### PR TITLE
Add Kisswood

### PR DESCRIPTION
**Pull request by elfing44**

### CODE REVIEW

File: plugins/french/kisswood.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+    return novels;
+  }
+
+  async fetchImage(url: string): Promise<string | undefined> {
+    // if your plugin has images and they won't load
+    // this is the function to fiddle with
+    return fetchFile(url);
+  }
+}
```
Comment (nyagami) [inline plugins/french/kisswood.ts:255]:
```suggestion
```

**Comment by nyagami** [comment]
oh sorry,  the suggestion includes class close  bracket, pls add it back

**Comment by nyagami** [comment]
this novel name get wrong name
image

**Comment by elfing44** [comment]
> this novel name get wrong name image

it's corrected

**Pull request merged**